dandiapi/api/mail.py

```diff
@@ -101,12 +101,15 @@ def registered_message(user, socialaccount):
 
 Welcome to DANDI.
 
-You are now registered on the DANDI archive. Registering allows you to create Dandisets and upload
-data right away. You can also use the Jupyterhub (https://hub.dandiarchive.org) for computing on
-dandisets in the cloud.
+You are now registered on the DANDI archive!
 
-It may take up to 24 hours for your hub account to be activated and for your email to be registered
-with our Slack workspace.
+It may take up to 24 hours for your account to be approved and activated.  During the approval
```

Comment:
I used to put a double space after ending sentences but a few years ago I learned that it's no longer the prevailing practice.

```suggestion
It may take up to 24 hours for your account to be approved and activated. During the approval
```

Reply:
"not prevailing" doesn't mean "not good" ;) Is it recommended for some specific reasons(s) to not do that?
Note: this is a text version of the email and IMHO double spacing helps to visually parse text.  HTML version will not have that

Reply:
The problem is, because it's "not prevailing", most rendering engines won't treat them the way typewriters did (i.e., present an extra space to make visual processing faster). I've seen things like a the double space getting "broken" across the ends of lines, etc.

For stuff like this, I like to just go with the prevailing notion of correctness (sort of like using Python Black to format source files) but honestly I don't have very strong feelings about this particular instance. If you want to keep it, that's ok with me--just resolve this thread so it's out of sight/out of mind 😸

Reply:
ping @yarikoptic